testcases/InstallUpstreamKernel.py

```diff
@@ -128,8 +128,7 @@ def is_url(path):
             time.sleep(10)
             if not self.use_kexec:
                 # FIXME: Handle distributions which do not support grub
-                con.run_command(
-                    "grub2-mkconfig  --output=/boot/grub2/grub.cfg")
+                con.run_command("grubby --set-default /boot/vmlinuz-%s" % res[-1])
```

Comment:
We will need code to handle vmlinux vs vmlinuz. Additionally we should also handle different distro flavours.